Car/api-service/Axios.tsx

```diff
@@ -32,10 +32,13 @@ Axios.interceptors.response.use(
     },
 
     async (error?: AxiosError) => {
-        if (axios.isAxiosError(error)) {
+        if (axios.isCancel(error)) {
+            console.log("catch cancel");
```

Comment:
is it ok to commit console logs?

Reply:
It's temporary. In next sprint I will change consol to azure application insigths.